packages/nocodb-sdk/src/lib/enums.ts

```diff
@@ -247,6 +247,14 @@ export const OrderedWorkspaceRoles = [
   null,
 ];
 
+export const OrderedOrgRoles = [
+  OrgUserRoles.SUPER_ADMIN,
+  OrgUserRoles.CREATOR,
+  OrgUserRoles.VIEWER,
+  // placeholder for no access
+  null,
```

Comment:
what about `OrgUserRoles.NO_ACCESS`?

Reply:
We only have 3 roles at Org level `Super`, `Creator` & `Viewer`

Reply:
I'm wondering why we need the null value here.

Reply:
I followed the same approch that already used for `OrderedWorkspaceRoles`

```ts
export const OrderedWorkspaceRoles = [
  WorkspaceUserRoles.OWNER,
  WorkspaceUserRoles.CREATOR,
  WorkspaceUserRoles.EDITOR,
  WorkspaceUserRoles.COMMENTER,
  WorkspaceUserRoles.VIEWER,
  // placeholder for no access
  null,
];
```

Reply:
@mertmit any idea why we used null as a placeholder for no access in `OrderedWorkspaceRoles `

Reply:
Initially we were depending OrderedWorkspaceRoles & OrderedProjectRoles to map roles between ws - project. Later as this approach was error prone I've started using WorkspaceRolesToProjectRoles instead, so it should be safe to get rid of that null in OrderedWorkspaceRoles, and it is not required at all for org level.

Reply:
Okay thanks, I will remove null form both places